apps/app/src/app/pages/pages.component.html

```diff
@@ -21,16 +21,27 @@
         routerLinkActive="active"
         >{{ link.label }}</a
       >
-      <div class="nav-divider" *ngIf="isSeqEnabled || isGrafanaEnabled"></div>
-      <a clrVerticalNavLink href="/seq/" target="_blank" *ngIf="isSeqEnabled">
+      <div
+        class="nav-divider"
+        *ngIf="
+          (settingsService.seqSubject$ | async) ||
```

Comment:
Tu peux éviter une double souscription en utilisant un petit trick sur le *ngIf. (à lire juste là https://medium.com/@yurykatkov/how-to-avoid-multiple-async-pipes-in-angular-ff0d51a8d368)